codalab/apps/chahub/utils.py

```diff
@@ -29,6 +29,9 @@ def send_to_chahub(endpoint, data, update=False):
                 'X-CHAHUB-API-KEY': settings.CHAHUB_API_KEY,
             }
         }
-        return requests.post(**kwargs)
+        if update:
+            return requests.patch(**kwargs)
+        else:
+            return requests.post(**kwargs)
```

Comment:
doesn't look like your data is a list, so I think that's gotta be the issue.

Reply:
good catch